kotunil/build.gradle.kts

```diff
@@ -6,93 +6,35 @@ import java.util.*
 version = project.extra["kotunil-version"]!!
 
 plugins {
-    kotlin("multiplatform") version "1.9.10"
+    alias(libs.plugins.kotlinMultiplatform)
     id("maven-publish")
     id("signing")
-    id("org.jetbrains.dokka") version "1.8.10"
+    id("org.jetbrains.dokka")
 }
 
-dependencies {
-    project(":kotunil-generators")
-}
-
-val docsDir = "build/docs"
-
 kotlin {
-    jvm {
-        compilations.all {
-            kotlinOptions.jvmTarget = "1.8"
-        }
-        testRuns["test"].executionTask.configure {
-            useJUnit()
-        }
-    }
-    mingwX64()
-    linuxX64()
-    macosX64()
-    macosArm64()
+    targetHierarchy.default()
```

Comment:
According https://stackoverflow.com/questions/77294118/im-getting-this-error-in-kmm-cannot-add-a-kotlinsourceset-with-name-iosmain we don't need it more

Reply:
Just adding current docs to hierarchies https://kotlinlang.org/docs/multiplatform-hierarchy.html#default-hierarchy